.travis.yml

```diff
@@ -22,4 +22,5 @@ script:
   - cargo test --workspace --all-targets -- --nocapture
   - cargo bench --workspace --all-targets
   - cargo run --example asynchronous
+  - cargo run --example get_started
```

Comment:
I think it will fail because of no jaeger agent right there.

Reply:
OK, I have removed this command.